graaljs/graaljs-maven-webpack-qrcode/src/test/java/com/example/AppTest.java

```diff
@@ -0,0 +1,19 @@
+package com.example;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest {
+
+    /**
+     * Rigorous Test :-)
+     */
+    @Test
+    public void shouldAnswerWithTrue() {
+        assertTrue(true);
```

Comment:
I understand that we want to keep the example as simple as possible, but still, if we include a test then it should at least execute something from the application. So, what about adding something like
```
App.main(new String[] { "should not throw" });
```